src/graphql/mutations/start_game.ts

```diff
@@ -82,7 +82,7 @@ async function validateGame(game: Models.Game, t): Promise<ValidateGameResult> {
       let unit = units.find(unit => unit.id == playerUnit.unitId);
       if (!unit) throw new Error(`PlayerUnit ${playerUnit.id} for player ${gamePlayer.player.name} references a non-existent Unit`);
 
-      totalPoints += unit.pointsCost();
+      totalPoints += unit.pointsCost;
```

Comment:
`pointsCost` is now just a property of a unit pulled straight from a db column rather than a calculated value from a function.